source/backend/api/Models/Concepts/Lease/LeaseMap.cs

```diff
@@ -0,0 +1,95 @@
+using Mapster;
+using Pims.Api.Helpers.Extensions;
+using Pims.Dal.Helpers.Extensions;
+using Entity = Pims.Dal.Entities;
+using Model = Pims.Api.Models.Concepts;
+
+namespace Pims.Api.Models.Concepts
+{
+    public class LeaseMap : IRegister
+    {
+        public void Register(TypeAdapterConfig config)
+        {
+            config.NewConfig<Entity.PimsLease, Model.LeaseModel>()
+                .Map(dest => dest.Id, src => src.LeaseId)
+                .Map(dest => dest.RowVersion, src => src.ConcurrencyControlNumber)
+                .Map(dest => dest.Amount, src => src.LeaseAmount)
+                .Map(dest => dest.RenewalCount, src => src.PimsLeaseTerms.Count)
+                .Map(dest => dest.Properties, src => src.PimsPropertyLeases)
+                .Map(dest => dest.LFileNo, src => src.LFileNo)
+                .Map(dest => dest.TfaFileNumber, src => src.TfaFileNumber)
+                .Map(dest => dest.PsFileNo, src => src.PsFileNo)
+                .Map(dest => dest.MotiName, src => src.GetMotiName())
+                .Map(dest => dest.ExpiryDate, src => src.GetExpiryDate())
+                .Map(dest => dest.StartDate, src => src.OrigStartDate)
+                .Map(dest => dest.ProgramName, src => src.GetProgramName())
+                .Map(dest => dest.OtherCategoryType, src => src.LeaseCategoryOtherDesc)
+                .Map(dest => dest.OtherProgramType, src => src.OtherLeaseProgramType)
+                .Map(dest => dest.OtherPurposeType, src => src.OtherLeasePurposeType)
+                .Map(dest => dest.OtherPurposeType, src => src.LeasePurposeOtherDesc)
```

Comment:
could you please confirm these two mappings are correct? they look duplicates to me...

```cs
  .Map(dest => dest.OtherPurposeType, src => src....)
  .Map(dest => dest.OtherPurposeType, src => src....)
```

Reply:
Looks like there are actually two database fields referring to the same logical field from the API. I've updated this to write to both and read from the newer field or if not set the older field.